### PR TITLE
Improve fullscreen version

### PR DESCRIPTION
**Pull request by johncordeiro**
**Proposed changes**:
- Full screen version with higher texts and brighter content

Captura de Tela 2020-03-28 às 1 48 52 PM

### CODE REVIEW

**Comment by johncordeiro** [comment]
Design Review:

- Title on the top is not proportional to the page;
- The animation on the text typing could be customizable;

**Comment by johncordeiro** [comment]
Bugs: 

- Line break in long devices:
Captura de Tela 2020-03-28 às 1 44 12 PM

File: dev/src/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1 class="test"> test</h1>
 <script src="index.js"  type="text/javascript"></script>
 <div id="webchat"></div>
 <script>
-  WebChat.default.init({
+  /*WebChat.default.init({
```
Comment (johncordeiro) [inline dev/src/index.html]: Please remove commented version of this code.

File: src/components/Widget/components/Conversation/components/Messages/components/Message/styles.scss [hunk 2]
```diff
@@ -2,36 +2,39 @@
...
       p {
-        margin: 0;
+        margin: 0 auto;
       }
     }
   }
-  
+
+
+
```
Comment (johncordeiro) [inline src/components/Widget/components/Conversation/components/Messages/components/Message/styles.scss]: 3 line breaks, is that correct?

File: src/components/Widget/components/Conversation/components/Messages/styles.scss [hunk 3]
```diff
@@ -94,6 +113,11 @@ div#wave {
 	height:13px;
 	margin-left: auto;
 	margin-right: auto;
+  .customText {
```
Comment (johncordeiro) [inline src/components/Widget/components/Conversation/components/Messages/styles.scss]: please fix this indentation

File: src/components/Widget/components/Conversation/components/Messages/styles.scss [hunk 4]
```diff
@@ -78,12 +85,24 @@
...
+      margin: 0.5em 0 0 -1em;
+      padding: 0;
+    }
   }
 }
 
 @media screen and (max-width: 800px)  {
   .messages-container {
     @include messages-container-fs;
+    padding: 3% 4% 0 4%!important;
```
Comment (johncordeiro) [inline src/components/Widget/components/Conversation/components/Messages/styles.scss]: why using `!important?`

File: src/components/Widget/index.js [hunk 5]
```diff
@@ -55,6 +55,7 @@ class Widget extends Component {
     const { connectOn, autoClearCache, storage, dispatch, defaultHighlightAnimation } = this.props;
 
     // add the default highlight css to the document
+
```
Comment (johncordeiro) [inline src/components/Widget/index.js]: remove line break

File: src/components/Widget/index.js [hunk 6]
```diff
@@ -95,7 +96,7 @@ class Widget extends Component {
       dispatch(showChat());
       dispatch(openChat());
     }
-    this.applyCustomStyle();
+    //this.applyCustomStyle();
```
Comment (johncordeiro) [inline src/components/Widget/index.js]: is this right? please remove the line if it's not used.

File: src/scss/common.scss [hunk 7]
```diff
@@ -33,10 +35,12 @@ $roboto: 'Roboto', serif;
   border-radius: 0;
   flex-shrink: 0;
   position: relative;
+  text-align: center;
 }
 
 @mixin title-fs {
-  //padding: 0 0 15px 0;
```
Comment (johncordeiro) [inline src/scss/common.scss]: Please remove commented line

File: src/scss/common.scss [hunk 8]
```diff
@@ -97,6 +126,52 @@ $roboto: 'Roboto', serif;
...
+
+  .sender a, .sender a:active, .sender a:hover {
+    display: inline;
+    position: absolute;
+    bottom: 1vh;
+    left: 1vh;
+    color: #696969;
+    outline: none;
+    text-decoration: none;
+
```
Comment (johncordeiro) [inline src/scss/common.scss]: remove unecessary line break

File: src/components/Widget/components/Conversation/components/Messages/styles.scss (hunk 4 above)
Comment (matheusenrique) [reply src/components/Widget/components/Conversation/components/Messages/styles.scss]: we use !important to guarantee that this param will not be overwrite by the padding on @include class

**Pull request merged**